tests/test_tiingo.py

```diff
@@ -131,3 +135,14 @@ def test_get_news_bulk_ids(self):
                 vcr.use_cassette('tests/fixtures/news_bulk_file_ids.yaml'):
             value = self._client.get_bulk_news()
             assert value
+
+    def test_get_news_as_objects(self):
+        """Fails because this API key lacks institutional license"""
+        with self.assertRaises(RestClientError):
```

Comment:
You can use the same cassettes that were used in the `test_get_news_articles` endpoint, and that will let these tests pass